corehq/apps/export/esaccessors.py

```diff
@@ -58,12 +60,16 @@ def get_ledger_section_entry_combinations(domain):
     """Get all section / entry combinations in a domain.
     :returns: a generator of namedtuples with fields ``section_id``, ``entry_id``, ``doc_count``
     """
-    terms = [
-        AggregationTerm('section_id', 'section_id'),
-        AggregationTerm('entry_id', 'entry_id'),
+    ledgers = LedgerAccessors(domain).get_ledger_values_for_cases()  # TODO this should get all ledgers
```

Comment:
Unfortunately this will be impossible to implement with an index only scan at the moment. need to think a bit on this, but I think it should be possible to figure out without resorting to ES.

Also @snopoke if ICDS has any exports with these "stock" columns, then I think there is a dependency on LedgerES